raiden-cli/src/routes/userDeposit.ts

```diff
@@ -109,8 +109,12 @@ async function determineAndExecuteRequestedInteraction(
 async function getUDCInfo(this: Cli, _request: Request, response: Response): Promise<void> {
   const balance = await this.raiden.getUDCCapacity();
   const totalDeposit = await this.raiden.getUDCTotalDeposit();
+  const svtAddress = await this.raiden.userDepositTokenAddress();
```

Comment:
What does `svt` stands for?

Reply:
Service Token aka RDN. This was mostly an informative addition, I'll rename if it was not clear.

Reply:
:heavy_check_mark:

Reply:
❤️